spec/core_ext/object/namespace_spec.rb

```diff
@@ -19,26 +19,8 @@ module Eee; end
       expect(Aaa::Bbb::Ccc::Eee.in_namespace?("Aaa::Bbb")).to be_truthy
     end
 
-    it "Instance of Class" do
-      expect(Aaa::Bbb::Ccc::Ddd.new.in_namespace?(Aaa::Bbb)).to be_truthy
-    end
-
```

Comment:
Shouldn't `Object#in_namespace?` go away too? It's now broken (per this test removal), right?